tests/spread/plugins/waf/snaps/waf-hello/snap/snapcraft.yaml

```diff
@@ -1,12 +1,12 @@
-name: waf-with-configflags
-version: 0.1
+name: waf-hello
+version: '1.0'
 summary: test the waf plugin
 description: |
   This is a basic waf snap with configflags.
 confinement: strict
 
 apps:
-  waf-with-configflags:
+  waf-hello:
     command: usr/local/bin/myprogram
```

Comment:
awe, you didn't change this to hello! :-)

Reply:
Didn't feel like actually learning Waf :stuck_out_tongue: .